src/js/ripple/amount.js

```diff
@@ -1,12 +1,16 @@
+ 'use strict';
+
+ /*eslint-disable new-cap*/
```

Comment:
I think that's equivalent to `new Amount()`. Are you proposing that we get rid of `Amount.NaN()`?